From a2ec8df37f5dddc81592f3b911e0d7358990e8ae Mon Sep 17 00:00:00 2001
From: Robin Gong <yibin.gong@nxp.com>
Date: Thu, 14 Jan 2016 11:40:40 +0800
Subject: [PATCH 1520/1691] MLK-12256: firmware: imx: sdma: fix bugs in uart
 script code

commit a2ec8df37f5dddc81592f3b911e0d7358990e8ae from
git://git.freescale.com/imx/linux-2.6-imx.git imx_4.1.15_1.0.0_ga

Fix two below bugs in script:
  -- check error flag in FIFO data, thus dma will stop once the error happen
     when real HW error triggered by switch boundrate, overun,etc. Actrually,
     no need to check in script since UART driver will handle this.
  -- clear other status bit in USR1.

Signed-off-by: Robin Gong <yibin.gong@nxp.com>
---
 firmware/imx/sdma/sdma-imx6q.bin.ihex | 110 +++++++-------
 firmware/imx/sdma/sdma-imx7d.bin.ihex | 272 +++++++++++++++++-----------------
 2 files changed, 191 insertions(+), 191 deletions(-)
 mode change 100644 => 100755 firmware/imx/sdma/sdma-imx6q.bin.ihex

diff --git a/firmware/imx/sdma/sdma-imx6q.bin.ihex b/firmware/imx/sdma/sdma-imx6q.bin.ihex
old mode 100644
new mode 100755
index 112bec4..d296f8f
--- a/firmware/imx/sdma/sdma-imx6q.bin.ihex
+++ b/firmware/imx/sdma/sdma-imx6q.bin.ihex
@@ -1,15 +1,15 @@
 :1000000053444D4103000000030000001C000000A9
-:1000100029000000C0000000D80700008202000094
+:1000100029000000C0000000D40700008202000098
 :10002000FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFE0
 :10003000FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFD0
 :10004000FFFFFFFFFFFFFFFF421A0000FFFFFFFF60
-:10005000EB020000B4180000FFFFFFFFA01A000031
+:10005000EB020000B4180000FFFFFFFF9E1A000033
 :10006000FFFFFFFFC0030000FFFFFFFFFFFFFFFFD9
 :10007000FFFFFFFFAB020000FFFFFFFF7B0300005D
 :10008000FFFFFFFFFFFFFFFF4C0400006E040000B6
 :10009000FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF70
 :1000A000FFFFFFFF001800005B180000EE190000C2
-:1000B0004F1B0000EA1A0000861B00000018000019
+:1000B0004D1B0000E81A0000841B0000001800001F
 :1000C000E3C1DB57F352016AFB52D36AFB521C1A9D
 :1000D000C36AE8621102FF3A3008D0023F7C8F0009
 :1000E0003F00D500017D8D00A005EB5D7804037D08
@@ -84,56 +84,56 @@
 :10053000DB57004D057DFAC1DB57F39977C25404B0
 :100540000AC2F099E3C1DB57F352056AFB568E02EB
 :10055000941AC36AC8626902267D941EC36ED36E64
-:10056000C8624802C86A9426981EC36EC86298265C
-:10057000C36E60020E7D981EC36EC8626C02037D5E
-:100580009826C36E839A4C02D36EC86A9826C36EAF
-:10059000939AC8626E02017C8D9A096A1A7F0125BE
-:1005A000004D217D549AE36E8F00D805017D8D00AA
-:1005B000C8626E02127D096A0C7F01250120F87C59
-:1005C000DB57004D107D286A047F0000FAC1DB571D
-:1005D000469A070004620C6A909A286AFA7F0462BD
-:1005E0007AC258045404FF081100FF18BC00CD0063
-:1005F000017C839A286AED7F04627AC20AC2439A18
-:10060000D9C1E3C1DB57F352056AFB568E02941A37
-:10061000025269021D7D941E06524802065A942613
-:10062000981E06524C02065A9826981E06526002E0
-:100630000A7C982606526E02237D096A1D7F0125D9
-:10064000004D247DB59A286A177F04627AC2E69A23
-:100650008F00D805017D8D00A00506526E02107D29
-:10066000096A0A7F0120F97C286A067F0000004D94
-:100670000D7DFAC1DB57A59A070004620C6AE39A64
-:10068000286AFA7F04627AC258045404286AF47F04
-:100690000AC2A29A2EDBDB57F352056AC7698F00A4
-:1006A000D500017D8D00A0057804037D79041C7DB3
-:1006B000157C79041E7CEE56C862287E6006027D99
-:1006C00010021202096A227F1202096A1F7F1202B7
-:1006D000096A1C7F03200048EF7C1D9B1500150054
-:1006E0000278C8620B6A1C9B15000278C8620A6A0D
-:1006F0001C9B0278C862096A097C286A077F004D42
-:10070000077DD352010802580004EE9A77C25404C0
-:10071000D35201080258010446DBEB9A700B110317
-:100720001353DB5F0A07D3588B00FB5E3CC1157D7A
-:10073000C05AC85B4EC1117C8803F05DFF0D1105E6
-:10074000FF1DBC05004D047D459B0807DB5F0A07C4
-:100750008B00C352CB5359C1EC7D0002369B016E16
-:100760000B612F7E0B622D7E0B632B7E0C0D17040D
-:10077000170417049D04081DCC05017C0C0DD16ADB
-:10078000000F4207C86FDD6F1C7F8E009D0001685F
-:100790000B67177ED56B04080278C86F1207117CAF
-:1007A0000B670F7E04080278C86F12070A7CDD6FA2
-:1007B000087FD169010FC86FDD6F037F010100045D
-:1007C0005B9B0700FF680C6800025B9BCBDBDB5781
-:1007D000F352056AFB52C76A8F00D500017D8D0078
-:1007E000A0057804037D79041C7D157C79041E7CAA
-:1007F000EE56C862287E6006027D10021202096A67
-:10080000227F1202096A1F7F1202096A1C7F0320DD
-:100810000048EF7CBA9B150015000278C8620B6A8D
-:10082000B99B15000278C8620A6AB99B0278C8624F
-:10083000096A097C286A077F004D077DD3520108A9
-:10084000025800048A9B77C25404D352010802580C
-:100850000104E3DB879B700B11031353DB5F0A0773
-:10086000D3588B00FB5E3CC1157DC05AC85B4EC19E
-:10087000117C8803F05DFF0D1105FF1DBC05004DC7
-:10088000047DE29B0807DB5F0A078B00C352CB5352
-:0808900059C1EC7D0002D39B6D
+:10056000C862010A1102C86A9426981EC36EC86246
+:100570009826C36E60020F7D981EC36EC8626C021F
+:10058000037D9826C36E819A100A1102D36EC86A41
+:100590009826C36E919AC8626D9A096A187F0125E0
+:1005A000004D1F7D559AE36E8F00D805017D8D00AB
+:1005B000C862096A0C7F01250120FA7CDB57004DD7
+:1005C000107D286A047F0000FAC1DB57469A0700B5
+:1005D00004620C6A8E9A286AFA7F04627AC258040E
+:1005E0005404FF081100FF18BC00CD00017C819A63
+:1005F000286AED7F04627AC20AC2439AD9C1E3C174
+:10060000DB57F352056AFB568E02941A02526902B6
+:100610001D7D941E06524802065A9426981E0652C4
+:100620004C02065A9826981E065260020A7C9826AA
+:1006300006526E02237D096A1D7F0125004D247D2F
+:10064000B39A286A177F04627AC2E49A8F00D805A9
+:10065000017D8D00A00506526E02107D096A0A7F99
+:100660000120F97C286A067F0000004D0D7DFAC14B
+:10067000DB57A39A070004620C6AE19A286AFA7FA2
+:1006800004627AC258045404286AF47F0AC2A09A09
+:100690002CDBDB57F352056AC7698F00D500017D5B
+:1006A0008D00A0057804037D79041C7D157C7904F8
+:1006B0001E7CEE56C862287E6006027D1002120281
+:1006C000096A227F1202096A1F7F1202096A1C7FCF
+:1006D00003200048EF7C1B9B150015000278C862C0
+:1006E0000B6A1A9B15000278C8620A6A1A9B027884
+:1006F000C862096A097C286A077F004D077DD352CA
+:10070000010802580004EC9A77C25404D35201083D
+:100710000258010444DBE99A700B11031353DB5FA9
+:100720000A07D3588B00FB5E3CC1157DC05AC85BDD
+:100730004EC1117C8803F05DFF0D1105FF1DBC0546
+:10074000004D047D439B0807DB5F0A078B00C35203
+:10075000CB5359C1EC7D0002349B016E0B612F7E9F
+:100760000B622D7E0B632B7E0C0D170417041704F0
+:100770009D04081DCC05017C0C0DD16A000F4207B9
+:10078000C86FDD6F1C7F8E009D0001680B67177EB0
+:10079000D56B04080278C86F1207117C0B670F7EB7
+:1007A00004080278C86F12070A7CDD6F087FD169E0
+:1007B000010FC86FDD6F037F01010004599B070023
+:1007C000FF680C680002599BC9DBDB57F352056ACE
+:1007D000FB52C76A8F00D500017D8D00A00578040B
+:1007E000037D79041C7D157C79041E7CEE56C8625D
+:1007F000287E6006027D10021202096A227F120220
+:10080000096A1F7F1202096A1C7F03200048EF7CDF
+:10081000B89B150015000278C8620B6AB79B1500DB
+:100820000278C8620A6AB79B0278C862096A097CC2
+:10083000286A077F004D077DD35201080258000443
+:10084000889B77C25404D352010802580104E1DBAB
+:10085000859B700B11031353DB5F0A07D3588B0082
+:10086000FB5E3CC1157DC05AC85B4EC1117C88033C
+:10087000F05DFF0D1105FF1DBC05004D047DE09BE3
+:100880000807DB5F0A078B00C352CB5359C1EC7DCD
+:040890000002D19BF6
 :00000001FF
diff --git a/firmware/imx/sdma/sdma-imx7d.bin.ihex b/firmware/imx/sdma/sdma-imx7d.bin.ihex
index b76eaef..4d88fe2 100644
--- a/firmware/imx/sdma/sdma-imx7d.bin.ihex
+++ b/firmware/imx/sdma/sdma-imx7d.bin.ihex
@@ -1,145 +1,145 @@
-:1000000053444D4104000000010000001C000000AA
-:100010002A000000C400000038080000840200002C
+:1000000053444D4104000000020000001C000000A9
+:100010002A000000C40000003A080000840200002A
 :10002000FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFE0
 :10003000FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFD0
-:10004000FFFFFFFFFFFFFFFF051A0000FFFFFFFF9D
-:10005000ED020000B1180000FFFFFFFF631A00006F
+:10004000FFFFFFFFFFFFFFFF081A0000FFFFFFFF9A
+:10005000ED020000B4180000FFFFFFFF641A00006B
 :10006000FFFFFFFFC2030000FFFFFFFFFFFFFFFFD7
 :10007000FFFFFFFFAD020000FFFFFFFF7D03000059
 :10008000FFFFFFFFFFFFFFFF4E04000070040000B2
 :10009000FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF70
-:1000A000FFFFFFFF0018000058180000B119000002
-:1000B000121B0000AD1A0000491B000000180000D0
+:1000A000FFFFFFFF001800005B180000B4190000FC
+:1000B000131B0000AE1A00004A1B000000180000CD
 :1000C000FFFFFFFFE5C1DB57F352016AFB52D36A22
 :1000D000FB521C1AC36AE8621102FF3A3008D002D0
-:1000E000337C8F00D500017D8D00A005EB5D780489
-:1000F000037D79041C7D207C7904157CEE56600616
-:10010000057D0965317E0A622F7E28980A622C7E61
-:1001100009652A7E12051205AD02C86A257F0320F3
-:100120000048ED7C409802780962C86A3F98150043
-:1001300002780A62C86A3F981500150002780B62BF
-:10014000C86A127CDF6D107FFB52041AC36AD36A3F
-:10015000C862042AC86AFB52D36A0000EB55004DFE
-:10016000077DFCC1DB5706980700CC680C6815C2F8
-:100170000CC20198D9C1E5C1DB57E35FE357F352E5
-:10018000216A8F00D500017D8D00A005EB5DFB5637
-:100190007804037D79042A7D317C7904207C700BFE
-:1001A0001103EB53000F6003057D0965377E0A627A
-:1001B000357E7C980A62327E0965307E1205120512
-:1001C000AD026007027C065A8498265A277F011FD9
-:1001D00003200048E87C700B11031353A598150009
-:1001E00004780962065A0962265AA4981500150077
-:1001F00004780A62065A0A62265AA4981500150065
-:10020000150004780B62065A0B62265A077C000020
-:10021000EB55004D067DFCC1E3575F9807000C6865
-:1002200015C20CC25C98700B110313536C07017C50
-:10023000D9C1FB5E8A066B07017CD9C1F35EDB592D
-:10024000D3588F0110010F398B003CC12B7DC05A50
-:10025000C85B4EC1277C88038906E35CFF0D11054E
-:10026000FF1DBC053E07004D187D700811007E077C
-:10027000097D7D07027D2852DC98F852DB54BC02D0
-:10028000CC02097C7C07027D2852E598F852D354B1
-:10029000BC02CC02097D0004D3988B00C052C85325
-:1002A00059C1D67D0002C398FF08BF007F07157DA6
-:1002B0008804D500017D8D00A005EB5D8F02120240
-:1002C0001202FF3ADA05027C3E071099A402DD0211
-:1002D000027D3E0710995E071099EB55004D027C98
-:1002E0009805EB5DF352FB546A07267D6C07017D90
-:1002F0004D996B07577C6907047D6807027D010EE5
-:1003000027999358D600017D8E009355A005935DE3
-:10031000A00602780255045D1D7C004E087C69072A
-:10032000037D0255177E3499045D147F890693502E
-:100330000048017D1F999899150006780255045DC3
-:100340004F070255245D2F07017C989917006F070E
-:10035000017C012093559D0007009FD9EB98D36C39
-:100360006907047D6807027D010E5C999358D600E9
-:10037000017D8E009355A005935DA006027802557D
-:10038000C86D0F7C004E087C6907037D0255097E0D
-:100390006999C86D067F890693500048017D54997C
-:1003A00098999299C36A6907047D6807027D010ED6
-:1003B0007F999358D600017D8E009355A005935DDB
-:1003C000A0060278C865045D0F7C004E087C6907B2
-:1003D000037DC865097E8C99045D067F890693506C
-:1003E0000048017D7799989993559D000700FF6C0F
-:1003F0009FD9EB980000E354EB55004D017CEB983E
-:10040000D398E354EB55FF0A1102FF1A7F07027CD1
-:10041000A005AC999D008C05BA05A0051002BA0490
-:10042000AD0454040600D9C1E5C1DB57F352056A97
-:100430008F00D500017D8D00A005FB567804037D5B
-:100440007904297D1F7C79042E7CE35D700D1105F4
-:10045000ED55000F6007027D0652CD992652337E7E
-:100460006005027D10021202096A2D7F1202096ADC
-:100470002A7F1202096A277F011F03200048EA7CB5
-:10048000E355F899150015001500047806520B6A1B
-:1004900026520B6AF79915001500047806520A6A6D
-:1004A00026520A6AF799150004780652096A2652FC
-:1004B000096A097C286A077F0000DB57004D057D2B
-:1004C000FCC1DB57B69979C254040CC2B399E5C19B
-:1004D000DB57F352056AFB568E02941AC36AC86250
-:1004E0006902267D941EC36ED36EC8624802C86A34
-:1004F0009426981EC36EC8629826C36E60020E7D55
-:10050000981EC36EC8626C02037D9826C36E469A1D
-:100510004C02D36EC86A9826C36E569AC8626E02A1
-:10052000017C509A096A1A7F0125004D217D179A96
-:10053000E36E8F00D805017D8D00C8626E02127DCA
-:10054000096A0C7F01250120F87CDB57004D107DE6
-:10055000286A047F0000FCC1DB57099A0700046287
-:100560000C6A539A286AFA7F04627CC258045404C5
-:10057000FF081100FF18BC00CD00017C469A286AD4
-:10058000ED7F04627CC20CC2069AD9C1E5C1DB577B
-:10059000F352056AFB568E02941A025269021D7DBF
-:1005A000941E06524802065A9426981E06524C0281
-:1005B000065A9826981E065260020A7C9826065211
-:1005C0006E02237D096A1D7F0125004D247D789AE6
-:1005D000286A177F04627CC2A99A8F00D805017D22
-:1005E0008D00A00506526E02107D096A0A7F012067
-:1005F000F97C286A067F0000004D0D7DFCC1DB57A9
-:10060000689A070004620C6AA69A286AFA7F046254
-:100610007CC258045404286AF47F0CC2659AF1DA4B
-:10062000DB57F352056AC7698F00D500017D8D0045
-:10063000A0057804037D79041C7D157C79041E7C5B
-:10064000EE56C862287E6006027D10021202096A18
-:10065000227F1202096A1F7F1202096A1C7F03208F
-:100660000048EF7CE09A150015000278C8620B6A1A
-:10067000DF9A15000278C8620A6ADF9A0278C862B7
-:10068000096A097C286A077F004D077DD35201085B
-:1006900002580004B19A79C25404D3520108025896
-:1006A000010409DBAE9A700B11031353DB5F0A07D9
-:1006B000D3588B00FB5E3CC1157DC05AC85B4EC150
-:1006C000117C8803F05DFF0D1105FF1DBC05004D79
-:1006D000047D089B0807DB5F0A078B00C352CB53DE
-:1006E00059C1EC7D0002F99A016E0B612F7E0B62FD
-:1006F0002D7E0B632B7E0C0D1704170417049D042D
-:10070000081DCC05017C0C0DD16A000F4207C86F93
-:10071000DD6F1C7F8E009D0001680B67177ED56B17
-:1007200004080278C86F1207117C0B670F7E04085B
-:100730000278C86F12070A7CDD6F087FD169010F4C
-:10074000C86FDD6F037F010100041E9B0700FF6877
-:100750000C6800021E9B8EDBDB57F352056AFB52CE
-:10076000C76A8F00D500017D8D00A0057804037D48
-:1007700079041C7D157C79041E7CEE56C862287EA7
-:100780006006027D10021202096A227F1202096AC3
-:100790001F7F1202096A1C7F03200048EF7C7D9BAB
-:1007A000150015000278C8620B6A7C9B1500027860
-:1007B000C8620A6A7C9B0278C862096A097C286A56
-:1007C000077F004D077DD3520108025800044D9B5E
-:1007D00079C25404D352010802580104A6DB4A9B93
-:1007E000700B11031353DB5F0A07D3588B00FB5EBA
-:1007F0003CC1157DC05AC85B4EC1117C8803F05DB9
-:10080000FF0D1105FF1DBC05004D047DA59B0807CC
-:10081000DB5F0A078B00C352CB5359C1EC7D00024A
-:10082000969BFBDBDB57F352016AFB52D36A8F00C6
-:10083000D500017D8D00A005EB5D7804037D790472
-:100840001C7D207C7904157CEE566006057D0965CB
-:100850002A7E0A62287ECE9B0A62257E0965237E57
-:1008600012051205AD02C86A1E7F03200048ED7C08
-:10087000E69B02780962C86AE59B150002780A6265
-:10088000C86AE59B1500150002780B62C86A0B7CEC
-:10089000DF6D097F0000EB55004D097DD352010843
-:1008A00002580004B59B0700CC680C6815C2D352EF
-:1008B00001080258010413DCB09B700B11031353A1
-:1008C000DB5F0A07D3588B00FB5E3CC1157DC05A25
-:1008D000C85B4EC1117C8803F05DFF0D1105FF1D43
-:1008E000BC05004D047D129C0807DB5F0A078B00E6
-:0C08F000C352CB5359C1EC7D0002039CA5
+:1000E0003F7C8F003F00D500017D8D00A005EB5DBA
+:1000F0007804037D79041C7D207C7904157CEE5600
+:100100006006057D0965337E0A62317E29980A62A0
+:100110002E7E09652C7E12051205AD02C86A277F66
+:1001200003200048ED7C419802780962C86A409833
+:10013000150002780A62C86A409815001500027816
+:100140000B62C86A147CDF6D127F7F07097DFB524A
+:10015000041AC36AD36AC862042AC86AFB52D36A03
+:100160000000EB55004D077DFCC1DB5706980700EA
+:10017000CC680C6815C20CC20198D9C1E5C1DB5727
+:10018000E35FE357F352216A8F00D500017D8D00B4
+:10019000A005EB5DFB567804037D79042A7D317C54
+:1001A0007904207C700B1103EB53000F6003057D75
+:1001B0000965377E0A62357E7F980A62327E09655C
+:1001C000307E12051205AD026007027C065A879840
+:1001D000265A277F011F03200048E87C700B11037B
+:1001E0001353A898150004780962065A0962265A22
+:1001F000A7981500150004780A62065A0A62265A62
+:10020000A79815001500150004780B62065A0B62BA
+:10021000265A077C0000EB55004D067DFCC1E357D4
+:10022000629807000C6815C20CC25F98700B11032E
+:1002300013536C07017CD9C1FB5E8A066B07017CF6
+:10024000D9C1F35EDB59D3588F0110010F398B00F0
+:100250003CC12B7DC05AC85B4EC1277C88038906F0
+:10026000E35CFF0D1105FF1DBC053E07004D187D29
+:10027000700811007E07097D7D07027D2852DF98F6
+:10028000F852DB54BC02CC02097C7C07027D285268
+:10029000E898F852D354BC02CC02097D0004D698E9
+:1002A0008B00C052C85359C1D67D0002C698FF08C2
+:1002B000BF007F07157D8804D500017D8D00A00556
+:1002C000EB5D8F0212021202FF3ADA05027C3E0752
+:1002D0001399A402DD02027D3E0713995E0713996C
+:1002E000EB55004D027C9805EB5DF352FB546A0719
+:1002F000267D6C07017D50996B07577C6907047D4B
+:100300006807027D010E2A999358D600017D8E0060
+:100310009355A005935DA00602780255045D1D7CEF
+:10032000004E087C6907037D0255177E3799045DEE
+:10033000147F890693500048017D22999B991500EE
+:1003400006780255045D4F070255245D2F07017C96
+:100350009B9917006F07017C012093559D000700B2
+:10036000A2D9EE98D36C6907047D6807027D010E5F
+:100370005F999358D600017D8E009355A005935D3B
+:10038000A00602780255C86D0F7C004E087C6907F4
+:10039000037D0255097E6C99C86D067F89069350CE
+:1003A0000048017D57999B999599C36A6907047D17
+:1003B0006807027D010E82999358D600017D8E0058
+:1003C0009355A005935DA0060278C865045D0F7C77
+:1003D000004E087C6907037DC865097E8F99045D1E
+:1003E000067F890693500048017D7A999B99935521
+:1003F0009D000700FF6CA2D9EE980000E354EB5576
+:10040000004D017CEE98D698E354EB55FF0A11029B
+:10041000FF1A7F07027CA005AF999D008C05BA05E5
+:10042000A0051002BA04AD0454040600D9C1E5C108
+:10043000DB57F352056A8F00D500017D8D00A005C2
+:10044000FB567804037D7904297D1F7C79042E7C7A
+:10045000E35D700D1105ED55000F6007027D06523A
+:10046000D0992652337E6005027D10021202096A7D
+:100470002D7F1202096A2A7F1202096A277F011F53
+:1004800003200048EA7CE355FB9915001500150090
+:10049000047806520B6A26520B6AFA991500150069
+:1004A000047806520A6A26520A6AFA9915000478F4
+:1004B0000652096A2652096A097C286A077F0000E9
+:1004C000DB57004D057DFCC1DB57B99979C2540457
+:1004D0000CC2B699E5C1DB57F352056AFB568E0292
+:1004E000941AC36AC8626902267D941EC36ED36ED5
+:1004F000C862010A1102C86A9426981EC36EC862B7
+:100500009826C36E60020F7D981EC36EC8626C028F
+:10051000037D9826C36E479A100A1102D36EC86AEB
+:100520009826C36E579AC862339A096A187F0125C4
+:10053000004D1F7D1B9AE36E8F00D805017D8D0055
+:10054000C862096A0C7F01250120FA7CDB57004D47
+:10055000107D286A047F0000FCC1DB570C9A07005D
+:1005600004620C6A549A286AFA7F04627CC25804B6
+:100570005404FF081100FF18BC00CD00017C479A0D
+:10058000286AED7F04627CC20CC2099AD9C1E5C118
+:10059000DB57F352056AFB568E02941A0252690227
+:1005A0001D7D941E06524802065A9426981E065235
+:1005B0004C02065A9826981E065260020A7C98261B
+:1005C00006526E02237D096A1D7F0125004D247DA0
+:1005D000799A286A177F04627CC2AA9A8F00D8058C
+:1005E000017D8D00A00506526E02107D096A0A7F0A
+:1005F0000120F97C286A067F0000004D0D7DFCC1BA
+:10060000DB57699A070004620C6AA79A286AFA7F86
+:1006100004627CC258045404286AF47F0CC2669AAF
+:10062000F2DADB57F352056AC7698F00D500017D06
+:100630008D00A0057804037D79041C7D157C790468
+:100640001E7CEE56C862287E6006027D10021202F1
+:10065000096A227F1202096A1F7F1202096A1C7F3F
+:1006600003200048EF7CE19A150015000278C8626B
+:100670000B6AE09A15000278C8620A6AE09A02786A
+:10068000C862096A097C286A077F004D077DD3523A
+:10069000010802580004B29A79C25404D3520108E6
+:1006A000025801040ADBAF9A700B11031353DB5F8E
+:1006B0000A07D3588B00FB5E3CC1157DC05AC85B4E
+:1006C0004EC1117C8803F05DFF0D1105FF1DBC05B7
+:1006D000004D047D099B0807DB5F0A078B00C352AE
+:1006E000CB5359C1EC7D0002FA9A016E0B612F7E4B
+:1006F0000B622D7E0B632B7E0C0D17041704170461
+:100700009D04081DCC05017C0C0DD16A000F420729
+:10071000C86FDD6F1C7F8E009D0001680B67177E20
+:10072000D56B04080278C86F1207117C0B670F7E27
+:1007300004080278C86F12070A7CDD6F087FD16950
+:10074000010FC86FDD6F037F010100041F9B0700CD
+:10075000FF680C6800021F9B8FDBDB57F352056AB2
+:10076000FB52C76A8F00D500017D8D00A00578047B
+:10077000037D79041C7D157C79041E7CEE56C862CD
+:10078000287E6006027D10021202096A227F120290
+:10079000096A1F7F1202096A1C7F03200048EF7C50
+:1007A0007E9B150015000278C8620B6A7D9B1500C0
+:1007B0000278C8620A6A7D9B0278C862096A097C6D
+:1007C000286A077F004D077DD352010802580004B4
+:1007D0004E9B79C25404D352010802580104A7DB8E
+:1007E0004B9B700B11031353DB5F0A07D3588B002D
+:1007F000FB5E3CC1157DC05AC85B4EC1117C8803AD
+:10080000F05DFF0D1105FF1DBC05004D047DA69B8D
+:100810000807DB5F0A078B00C352CB5359C1EC7D3D
+:100820000002979BFCDBDB57F352016AFB52D36A51
+:100830008F00D500017D8D00A005EB5D7804037D60
+:1008400079041C7D207C7904157CEE566006057DBC
+:1008500009652A7E0A62287ECF9B0A62257E096589
+:10086000237E12051205AD02C86A1E7F03200048D0
+:10087000ED7CE79B02780962C86AE69B1500027866
+:100880000A62C86AE69B1500150002780B62C86A06
+:100890000B7CDF6D097F0000EB55004D097DD352C5
+:1008A000010802580004B69B0700CC680C6815C20A
+:1008B000D35201080258010414DCB19B700B1103E0
+:1008C0001353DB5F0A07D3588B00FB5E3CC1157DD9
+:1008D000C05AC85B4EC1117C8803F05DFF0D110545
+:1008E000FF1DBC05004D047D139C0807DB5F0A0754
+:0E08F0008B00C352CB5359C1EC7D0002049C17
 :00000001FF
-- 
1.9.1

